pkg/kubelet/dockertools/manager.go

```diff
@@ -873,101 +871,70 @@ func getDockerNetworkMode(container *dockertypes.ContainerJSON) string {
 }
 
 // dockerVersion implementes kubecontainer.Version interface by implementing
-// Compare() and String() (which is implemented by the underlying semver.Version)
-// TODO: this code is the same as rktVersion and may make sense to be moved to
-// somewhere shared.
-type dockerVersion struct {
-	*semver.Version
-}
-
-// Older versions of Docker could return non-semantically versioned values (distros like Fedora
-// included partial values such as 1.8.1.fc21 which is not semver). Force those values to be semver.
```

Comment:
Yeah, I removed it. https://github.com/kubernetes/kubernetes/pull/23907/commits/e64167b8940901e1e6b61dd313e3576c56fca587#diff-e2fb6215bbaa9210484ba6b85101007aL134